fdbserver/TLogServer.actor.cpp

```diff
@@ -1910,16 +1907,13 @@ Future<Void> tLogPeekMessages(PromiseType replyPromise,
 			// TODO (version vector) all tags may be too many, instead,  standard deviation?
 			wait(waitForMessagesForTag(logData, reqTag, reqBegin, SERVER_KNOBS->BLOCKING_PEEK_TIMEOUT));
 			double latency = now() - startTime;
-			if (logData->blockingPeekLatencies.find(reqTag) == logData->blockingPeekLatencies.end()) {
-				UID ssID = nondeterministicRandom()->randomUniqueID();
-				std::string s = "BlockingPeekLatencies-" + reqTag.toString();
-				logData->blockingPeekLatencies.try_emplace(reqTag,
-				                                           s,
-				                                           ssID,
-				                                           SERVER_KNOBS->LATENCY_METRICS_LOGGING_INTERVAL,
-				                                           SERVER_KNOBS->LATENCY_SKETCH_ACCURACY);
-			}
-			LatencySample& sample = logData->blockingPeekLatencies.at(reqTag);
+			auto [it, inserted] =
```

Comment:
```suggestion
			auto [it, _] =
```

But given this code is in an actor context, this may cause redefinition compiler errors. If so, we can keep the previous instance as well for code consistency. I'm fine either way, no strong preference.